docs/docs/how-to/Client-Side SDKs/ios-sdk.md

```diff
@@ -105,7 +123,24 @@ if (showFeature) {
 }
 ```
 
-### Step 4. Unit Testing (Optional)
+### Step 4. Track Events
+
+:::note
+The Objc SDK supports event tracking from version 2.0.2.
+:::
+
+
+The event tracking feature can record the actions taken by the user in the application as events.
+
+Events are related to toggle's metrics. For more information about event analysis, please read [Event Analysis](../../tutorials/analysis).
+
+```objective-c
+[fp trackWithEvent:@"YOUR_CUSTOM_EVENT_NAME"];
+// Providing a metric value to track
+[fp trackWithEvent:@"EventWithValue" value:2.0];
```

Comment:
Is `EventWithValue` or `YOUR_CUSTOM_EVENT_NAME` here?

Reply:
good catch